src/App.vue

```diff
@@ -0,0 +1,316 @@
+<template>
+    <v-app>
+        <v-main>
+            <div v-if="loading">
+                <v-progress-circular
+                    indeterminate
+                    color="primary"
+                />
+            </div>
+            <div v-else>
+                <visualization-overview
+                    :filter="0"
+                    :go-count-table-processor="goProcessor!"
+                    :go-ontology="goOntology!"
+                    :ec-count-table-processor="ecProcessor!"
+                    :ec-ontology="ecOntology!"
+                    :interpro-count-table-processor="interproProcessor!"
+                    :interpro-ontology="interproOntology!"
+                    :filter-id="2"
+                    :analysis-in-progress="loading"
+                    :show-percentage="true"
+                    :ec-tree="ecTree!"
+                    :ncbi-tree="ncbiTree!"
+                    :ncbi-ontology="ncbiOntology!"
+                    :ncbi-count-table-processor="lcaProcessor!"
+                />
+
+                <go-summary-card
+                    :analysis-in-progress="loading"
+                    :go-processor="goProcessor!"
+                    :go-ontology="goOntology!"
+                    :ncbi-processor="lcaProcessor!"
+                    :ncbi-tree="ncbiTree!"
+                    :show-percentage="true"
+                    :filter="5"
+                    @filter-percentage-change="() => {}"
+                />
+            </div>
+
+<!--            <div v-else>-->
+<!--                <single-peptide-summary-->
+<!--                    :go-link="true"-->
+<!--                    :assay="singlePeptideStatus!"-->
+<!--                    :ec-link="true"-->
+<!--                    :interpro-link="true"-->
+<!--                />-->
+<!--                <single-peptide-analysis-->
+<!--                    :assay="singlePeptideStatus!"-->
+<!--                />-->
+<!--            </div>-->
+
+<!--            <matched-proteins-table-->
+<!--                v-else-->
+<!--                :assay="singlePeptideStatus!"-->
+<!--            />-->
+<!--            <lineage-table-->
+<!--                v-else-->
+<!--                :assay="singlePeptideStatus!"-->
+<!--            />-->
```

Comment:
Aha, I left these in on purpose because they aid us in locally testing the Unipept Components in the future (without having to build and import the library constantly during testing in the Unipept Web Application).